src/pages/standard-report/add-edit-report/validate.js

```diff
@@ -0,0 +1,49 @@
+import { reportTypes } from '../standard.report.conf'
+
+export const validateNewReport = values => {
```

Comment:
There's nothing wrong with form-level validation, but I find that working with field-level validation has a few advantages:
- easier to reason about because you only have to consider one field at a time
- better composition, you can create a bunch of general purpose validators and compose them in your form, especially because a `<Field/>`s `validate` prop can take an array of validators too
[See an example here](https://codesandbox.io/s/2k054qp40)

Reply:
I thought about this as well. The reason I didn't do this is that some fields depend on other field values and the current way I don't need to create a new function on every render.